src/generate_daily_plot.R

```diff
@@ -26,15 +32,12 @@ generate_daily_plot <- function(plot_data_arg, df_list_arg, site_arg, parameter_
   }
 
   # determining the index for the site of interest.
-  if (site_arg %in% sites_order) {
+  if (site_arg %in% tributary_sites$site) {
```

Comment:
although it would be messy, I'm wondering if the best course of action is just a if statement for each site and then manually telling it which sites to look for?

ie if(site_arg == "archery virridy"){
select_sites <- c('boxelder', 'archery', 'river bluffs')
}
 if(site_arg == "river bluffs"){
select_sites <- c( 'archery',  "archery virrridy', 'river bluffs')
}
 if(site_arg == "spring creek"){
select_sites <- c(spring creek, prospect)

Reply:
I took a bit of a "hybrid" approach between Juan's current and your suggestion here in `network_check` - @juandlt-csu  you may want to take a look at what I put together for that!